experimenter/experimenter/glean/views.py

```diff
@@ -0,0 +1,34 @@
+from django.views.generic.edit import UpdateView
+
+from experimenter.glean.models import Prefs
+
+
+class OptOutView(UpdateView):
+    model = Prefs
+    fields = ["opt_out"]
+    template_name = "glean/opt_out_checkbox.html"
+
+    def get_object(self, queryset=None):
+        return self.request.user.glean_prefs
+
+    def form_valid(self, form):
+        if ("opt_out" in form.changed_data) and (form.cleaned_data["opt_out"] is True):
+            # send glean opt-out for request.user.id
+            pass
```

Comment:
@travis79 I assume this needs to send a deletion request when users opt-out, but I can't find any docs about how to do that, nor how to indicate that `nimbus.nimbus_user_id` is the key to delete on

Reply:
won't that disable data collection for the entire server, rather than for an individual user, since glean is running on the server side?